packages/@aws-cdk/lambda-layer-awscli/layer/Dockerfile

```diff
@@ -9,27 +9,19 @@ WORKDIR /tmp
 #
 
 RUN yum update -y \
-    && yum install -y zip unzip wget tar gzip python3
+    && yum install -y zip unzip wget tar gzip
 
 #
 # aws cli
 #
 
-ARG AWSCLI_VERSION=0.0.0
-
-RUN curl https://s3.amazonaws.com/aws-cli/awscli-bundle-${AWSCLI_VERSION}.zip -o awscli-bundle.zip
-RUN unzip awscli-bundle.zip
-RUN python3 ./awscli-bundle/install -i /opt/awscli -b /opt/awscli/aws
+COPY requirements.txt ./
+RUN python -m pip install -r requirements.txt -t /opt/awscli
 
 # organize for self-contained usage
-RUN mv /opt/awscli                                      /opt/awscli.tmp
-RUN pyver=$(python3 -c 'import sys; v = sys.version_info; print(f"{v[0]}.{v[1]}")') && \
-    mv /opt/awscli.tmp/lib/python${pyver}/site-packages /opt/awscli
```

Comment:
I think if we don't do this, your statement of "this will work with any Python 3.x" runtime is going to be true. It's probably going to need to be the exact same Python version as the one that we used to install this, i.e. 3.7.

I'm saying that because I'm pretty sure the disk layout is going to look like this:

```
/opt/awscli/lib/python3.7/awscli
```

Can you confirm that, and see if there's something to be done about that?

Reply:
@rix0rrr  The directory layout that is produced doesn't have that `lib` folder at all. I also don't think doing version manipulation in the build image will have an impact on the version of python in the runtime environment.

But anyway I ended up adding integration tests for Python 3.6, 3.7, & 3.9.